Application/EdFi.Ods.Api/ScheduledJobs/Configurators/SchedulerConfiguratorBase.cs

```diff
@@ -0,0 +1,66 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using EdFi.Ods.Common.Configuration;
+using log4net;
+using Quartz;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace EdFi.Ods.Api.ScheduledJobs.Configurators
+{
+    public abstract class SchedulerConfiguratorBase<T> : ISchedulerConfigurator
+        where T : IJob
+    {
+        private readonly ILog _logger = LogManager.GetLogger(typeof(SchedulerConfiguratorBase<T>));
+
+        public const string DefaultCronExpression = "0 0/30 * 1/1 * ? *"; // Run every 30 minutes repeatedly
+
+        public abstract string Name { get; }
+
+        public abstract JobBuilder GetJobBuilder();
```

Comment:
You can provide a default implementation like: 
`public virtual JobBuilder GetJobBuilder() => JobBuilder.Create<T>();`

This way, there is less code duplication for common scenarios, the user can still override it if necessary.

Reply:
Modified abstract function and removed overrides from concrete implementations